src/ImageService.Data/ImageRepository.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using LT.DigitalOffice.ImageService.Data.Interfaces;
+using LT.DigitalOffice.ImageService.Data.Provider;
+using LT.DigitalOffice.ImageService.Models.Db;
+using LT.DigitalOffice.ImageService.Models.Dto.Constants;
+using LT.DigitalOffice.Models.Broker.Enums;
+using Microsoft.EntityFrameworkCore;
+
+namespace LT.DigitalOffice.ImageService.Data
+{
+  public class ImageRepository : IImageRepository
+  {
+    private readonly IDataProvider _provider;
+
+    private string GetTargetDBTableName(ImageSource imageSource)
+    {
+      string tableName = String.Empty;
+
+      switch (imageSource)
+      {
+        case ImageSource.User: tableName = DBTablesNames.USER; break;
```

Comment:
May be make it more easier?
![image](https://user-images.githubusercontent.com/43515136/169139713-2fad719f-9766-4649-a257-390dce191126.png)

Reply:
done